test/e2e/framework/pod/utils.go

```diff
@@ -130,3 +137,82 @@ func GetRestrictedContainerSecurityContext() *v1.SecurityContext {
 		Capabilities:             &v1.Capabilities{Drop: []v1.Capability{"ALL"}},
 	}
 }
+
+var psaEvaluator, _ = psapolicy.NewEvaluator(psapolicy.DefaultChecks())
+
+// MustMixinRestrictedPodSecurity makes the given pod compliant with the restricted pod security level.
+// If doing so would overrwrite existing non-conformant configuration, a test failure is triggered.
+func MustMixinRestrictedPodSecruity(pod *v1.Pod) *v1.Pod {
```

Comment:
```suggestion
func MustMixinRestrictedPodSecurity(pod *v1.Pod) *v1.Pod {
```

Reply:
Ugh, this is what I get for overusing autocomplete. Thanks for catching it.